JSDemos/Demos/PivotGrid/RunningTotals/description.md

```diff
@@ -1 +1,3 @@
-The PivotGrid component is&nbsp;able to&nbsp;display running total values next to&nbsp;summary cells. The component can perform cross-group accumulation of&nbsp;running total values or&nbsp;accumulate them within each group separately.
\ No newline at end of file
+You can use the [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property of the to enable calculation of running totals in the PivotGrid. Specify this property in a field object in the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array (see the `Running Total` field in this demo). Assign *"column"* or *"row"* to this property to calculate running totals for columns or rows respectively.
```

Comment:
```suggestion
You can use the fields[].[runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to calculate running totals for a specific PivotGrid field. To do this by columns or rows, set this property to *"column"* or *"row"* respectively.
```

Reply:
Before you get to the properties, you should begin an introductory sentence to create more natural flow in the text.

```suggestion
The PivotGrid can calculate running totals by rows or columns. To specify the direction, set a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to *"row"* or *"column"*. This property is available only for data fields.
```